.prow/pull-core-pwa-test.yaml

```diff
@@ -0,0 +1,15 @@
+postsubmits:
+  - name: pull-core-pwa-test
+    run_if_changed: "^web/pwa"
```

Comment:
This probably needs to run if web/pwa, web/auth or web/app/client changed. I think there is another prow job with that kind of regex that you could use as inspiration

Reply:
Lol, now that we merged the lerna stuff, this probably still need to change ^^

Reply:
After all, I think that the pwa job can be removed, I believe that the changes to lerna basically test all frontend stuff in a single job